lib/msal-angular/src/public-api.ts

```diff
@@ -17,6 +17,5 @@ export { MsalInterceptorConfiguration, MsalInterceptorAuthRequest, ProtectedReso
 export { MSAL_INSTANCE, MSAL_GUARD_CONFIG, MSAL_INTERCEPTOR_CONFIG } from "./constants";
 export { MsalBroadcastService } from "./msal.broadcast.service";
 export { MsalModule } from "./msal.module";
-export { MsalRedirectComponent } from "./msal.redirect.component";
```

Comment:
FYI this would be a breaking change, which would be a non-starter for this type of PR.